docs/kubernetes-versions.md

```diff
@@ -23,7 +23,7 @@
 | v1.20.4   | :white_check_mark: |
 | v1.20.6   | :white_check_mark: |
 | v1.20.10   | :white_check_mark: |
-| v1.21.4   | :white_check_mark: |
+| v1.21.5   | :white_check_mark: |

```

Comment:
KubeKey also supports install Kubernetes `v1.21.4`. Maybe we should add `v1.21.5`,  rather than modify the `v1.21.4`.

Reply:
That's better

Reply:
That's  good

Reply:
@rockpanda So, could you please modify it?